src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, useCallback, useState, useEffect } from "react";
+import { Link } from "../../../../components/atoms/link";
+import Modal, { useModalButtonHandler } from "../../../../core/utils/modal";
+import { useText } from "../../../../core/utils/text";
+import DateInput from "./date-input";
+import { useUpdateV5 } from "../../../../core/fbs/fbs";
+import { PatronV5 } from "../../../../core/fbs/model";
+import { getModalIds } from "../../../../core/utils/helpers/general";
+import { useConfig } from "../../../../core/utils/config";
+
+interface PauseReservationProps {
+  id: string;
+  user: PatronV5;
+}
+
+const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
+  const t = useText();
+  const { mutate } = useUpdateV5();
+  const { close } = useModalButtonHandler();
+  const { pauseReservation } = getModalIds();
+  const config = useConfig();
+  const [startDate, setStartDate] = useState<string>(
+    (config("pauseReservationStartDateConfig") as string) || ""
```

Comment:
It should be possible to write:
```
config<string>(....

Reply:
Instead of 
```
...as string

Reply:
It is because of how this [function overloading](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/a9f8d508e91ac7fb8ffa6ade98b784210a7f2a01/src/core/utils/config.tsx#L5-L22) has been constructed.
I guess you can leave it as is. It would be nice with a todo before the line wit `as string` mentioneing that it would be nice to do it otherwise but then the overloading of the `config`function would need to be changed.